specification/communication/data-plane/Rooms/stable/2024-04-15/communicationservicesrooms.json

```
@@ -0,0 +1,643 @@
+          },
+          {
+            "in": "body",
+            "name": "updateParticipantsRequest",
+            "description": "An updated set of participants of the room.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateParticipantsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The participants were successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/UpdateParticipantsResult"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update participants": {
+            "$ref": "./examples/Participants_Update.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationError": {
+      "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationError"
+    },
+    "CommunicationErrorResponse": {
+      "$ref": "../../../Common/stable/2022-07-13/common.json#/definitions/CommunicationErrorResponse"
+    },
+    "CreateRoomRequest": {
+      "description": "Request payload for creating new room.",
+      "type": "object",
+      "properties": {
+        "validFrom": {
+          "format": "date-time",
+          "description": "The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. The default value is the current date time.",
+          "type": "string",
+          "example": "2021-09-07T07:55:41Z"
+        },
+        "validUntil": {
+          "format": "date-time",
+          "description": "The timestamp from when the room can no longer be joined. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. The default value is the current date time plus 180 days.",
+          "type": "string",
+          "example": "2021-09-08T15:55:41Z"
+        },
+        "pstnDialOutEnabled": {
+          "description": "Set this flag to true if, at the time of the call, dial out to a PSTN number is enabled in a particular room. By default, this flag is set to false.",
+          "type": "boolean"
+        },
+        "participants": {
+          "description": "(Optional) Participants to be invited to the room.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/ParticipantProperties"
+          },
+          "example": {
+            "rawId1": {
+              "role": "Presenter"
+            },
+            "rawId2": {
+              "role": "Attendee"
+            },
+            "rawId3": null
```

`null` is not a "ParticipantProperties" object so this part of the example is invalid. Probably best to remove it.
```suggestion
            }
```

---

Thank you Mike. We use `null ` to remove the entry in the PATCH operation. However, it does not make sense in the create scenario here. I am gong to remove it.